specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -523,930 +523,1019 @@
-                        },
-                        "startTime": {
-                            "readOnly": true,
-                            "type": "string",
-                            "description": "The start time of script action execution."
-                        },
-                        "endTime": {
-                            "readOnly": true,
-                            "type": "string",
-                            "description": "The end time of script action execution."
-                        },
-                        "status": {
-                            "readOnly": true,
-                            "type": "string",
-                            "description": "The current execution status of the script action."
-                        },
-                        "operation": {
-                            "readOnly": true,
-                            "type": "string",
-                            "description": "The reason why the script action was executed."
-                        },
-                        "executionSummary": {
-                            "readOnly": true,
-                            "type": "array",
-                            "items": {
-                                "$ref": "#/definitions/ScriptActionExecutionSummary"
-                            },
-                            "description": "The summary of script action execution result."
-                        },
-                        "debugInformation": {
-                            "readOnly": true,
-                            "type": "string",
-                            "description": "The script action execution debug information."
-                        }
-                    }
-                }
-            ],
-            "description": "The execution details of a script action."
+        "minInstanceCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum instance count of the cluster."
         },
-        "ClusterListRuntimeScriptActionDetailResult": {
-            "properties": {
-                "value": {
-                    "readOnly": true,
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/RuntimeScriptActionDetail"
-                    }
-                },
-                "nextLink": {
-                    "type": "string",
-                    "readOnly": true,
-                    "description": "The link (url) to the next page of results."
-                }
-            },
-            "description": "The list runtime script action detail response."
+        "targetInstanceCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The instance count of the cluster."
+        },
+        "autoscale": {
+          "$ref": "#/definitions/Autoscale",
+          "x-ms-client-name": "autoscaleConfiguration",
```

Comment:
This extension goes inside the schema definition of Autoscale.

Reply:
Refer to other class definitions in `Role`, Such as `virtualNetworkProfile`, `osProfile`,`HardwareProfile`, We can make this extension goes inside the schema definition of Autoscale. same as definitions  of `osProfile`,`HardwareProfile`

Reply:
For a property, referencing a definition, all the fields should be inside the definition being referenced. In this case, the extension should be inside Autoscale definition. We allow description to be at the same level, but just for documentation purposes.

Reply:
@NelsonDaniel We did an experiment：when we make the extension `"x-ms-client-name": "autoscaleConfiguration"` inside Autoscale definition, the generated code does not match what we expected，so we decide to keep the origin writing style.

Reply:
Got it. Solve merge conflicts. Apart from that LGTM